sdcm/k8s_configs/cluster.yaml

```diff
@@ -0,0 +1,102 @@
+# Namespace where the Scylla Cluster will be created
```

Comment:
doesn't we want to take those out of the operator source code ? and not duplicate them in SCT ? or we need to alter them anyhow ?

Reply:
It's kind of k8s boilerplate, and yes, we want to have more control on that stuff in SCT. At least for now.